build/module.cmake

```diff
@@ -106,6 +106,7 @@ target_include_directories(${MODULE} PUBLIC
     ${PROJECT_ROOT_DIR}/src/framework
     ${PROJECT_ROOT_DIR}/src/framework/global
     ${PROJECT_ROOT_DIR}/src/engraving
+    ${PROJECT_ROOT_DIR}/thirdparty/googletest/googletest/include/
```

Comment:
need last '/'?

Reply:
not important, but I will remove it for consistency)